quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java

```diff
@@ -259,6 +260,27 @@ void checkJdbcDriver(BuildProducer<JdbcDriverBuildItem> ignore) {
         }
     }
 
+    /**
+     * Parse the default configuration for the User Profile provider
+     */
+    @BuildStep
+    @Produce(UserProfileBuildItem.class)
```

Comment:
just a nit, can this be simplified as just returning, rather than producing, a UserProfileBuildItem?

Reply:
Yep, could be.